### PR TITLE
Fix the Compilation warnings

### PR DESCRIPTION
**Pull request by inforithmics**

### CODE REVIEW

**Comment by inforithmics** [comment]
This pull request reduces the Warnings in Appveyor from more than 200 to roughly 20 Warning messges.

File: Samples/Mapsui.Samples.Maui/Mapsui.Samples.Maui.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
+		<TargetFrameworks>net6.0-android31.0;net6.0-ios13.6;net6.0-maccatalyst13.5</TargetFrameworks>
```
Comment (inforithmics) [inline Samples/Mapsui.Samples.Maui/Mapsui.Samples.Maui.csproj:4]: By fixing the warnings I discovered that the sample project did not include the Target versions for Android,ios and maccatalyst. When I enabled this I could remove the Android,iOS, Maccatalyst and Windows Targets of Mapsui.Ui.Maui. Simplyfing and accelerating the Build

Comment (pauldendulk) [reply Samples/Mapsui.Samples.Maui/Mapsui.Samples.Maui.csproj:4]: These kind of simplifications are very important. A user will never ask for something like this, so we should take case of this ourselves.

**Review by janusw — APPROVED** [review]
Looks great. Thank you!

**Comment by inforithmics** [comment]
Merging now

**Pull request merged**